### PR TITLE
refactor/474 - Overhaul LOWW

### LINKED ISSUE #474: Airport Revival :: Flughafen Wien-Schwechat
Vienna International Airport (German: Flughafen Wien-Schwechat; IATA: VIE, ICAO: LOWW) is the international airport of Vienna, the capital of Austria, located in Schwechat, 18 km (11 mi) southeast of central Vienna and 57 km west of Bratislava. It is the country's largest airport and serves as the hub for Austrian Airlines and a base for Eurowings Europe. It is capable of handling wide-body aircraft up to the Airbus A380. The airport features a dense network of European destinations as well as long-haul flights to Asia, North America and Africa. In 2017, the airport handled 24.4 million passengers, a 4.5% increase compared to 2016.

- [x] basic airport information
- [ ] new terrain
- [ ] new airspace
- [ ] new fixes
- [ ] restricted areas
- [ ] airways
- [ ] SIDs
- [ ] STARs
- [ ] spawnPatterns
- [ ] videomap

### PR DESCRIPTION
**Pull request by JonasStuhlberger**
Resolves #474.
Replaces #518.

Add LOWW Vienna Schwechat International Airport to the sim.

Things that have been added/updated:

- **Base Airport Properties**: Departing runway 29, Arrival runway 34.
- **Airspace boundaries**: Have been updated according to AustroControl charts
- **Fixes**: Removed unneded/outdated waypoints inside of controlled airspace, added all VORs in the vicinity of Vienna.
- **Restricted Airspace**: All Restricted Airspaces and Areas have been added.
- **Runways**: ILS settings and elevations have been slightly edited.
- **Standard Procedures**: Added all SIDs, STARs and RNAV Transitions, including NAP (Noise Abatement Procedures).
- **Spawnpatterns**: Added the majority of airlines operating from Vienna, including cargo carriers. Some Airlines might be missing, these include charter carriers and airlines flying to Vienna only seasonally. Spawn rates have been calculated following Flightaware, projectFLY, wikipedia and other sources. Occasional GA aircraft (_fastGA_/Private Jets for the most part, GA props are a rarity in the sim, just like in real life).
- **Video Map**: A video map has been created following charts from SkyVector, AustroControl and vACC-Austria, also including VOR stations, ILS centerlines, sourrounding GA airports.
- **Terrain .geojson**: Is included (thanks @Marcel510 for creating this for me).
- **Airways**: No Airways inside the Austrian Airspace due to Free Route Airspace, some Airways included for Spawnpatterns in the FIRs of Munich and Prague.
- **Airspace Stratification**: Not supported by the sim at the moment, option for Stratification included in the airport filed
- **Airport Documentation/Guide**: Included.
- **Airlines**: Some Airlines have been added and several have been edited, to allow for realistic airfleets flying in and out of Vienna.

That should be it for the most part, if I did something wrong with this whole pull request thing, please contact me as soon as possible. And many thanks to all the people who helped me (@Marcel510, @erikquinn, @indianbhaji, @Fechulo and @nategeslin to name a few. Thanks guys! :) )

### CODE REVIEW

File: assets/airlines/pev.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "icao": "pev",
+    "name": "Peoples Viennaline",
```
Comment (Fechulo) [inline assets/airlines/pev.json]: They rebranded to just `People's` last May

File: assets/airlines/pev.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        "name": "Peoples",
+        "callsignFormats": [
+            "###"
+        ]
+    },
+    "fleets": {
+        "default": [
+            ["E170", 2]
+        ],
+        "e170": [
```
Comment (Fechulo) [inline assets/airlines/pev.json]: No need for this fleet since they only fly one aircraft

File: assets/airlines/rot.json [hunk 3]
```diff
@@ -18,6 +18,16 @@
         ],
         "a318": [
             ["A318", 4]
+        ],
+        "shortjet": [
```
Comment (Fechulo) [inline assets/airlines/rot.json]: use camelCase

File: assets/airlines/rot.json [hunk 4]
```diff
@@ -18,6 +18,16 @@
...
+            ["A318", 4],
+            ["B733", 4],
+            ["B737", 4],
+            ["B738", 4]
+        ],
+        "shortprop": [
```
Comment (Fechulo) [inline assets/airlines/rot.json]: use camelCase

File: assets/airlines/tap.json [hunk 5]
```diff
@@ -31,7 +31,7 @@
             ["A321", 4],
             ["E190", 13]
         ],
-        "airbusShort": [
+        "airbusshort": [
```
Comment (Fechulo) [inline assets/airlines/tap.json]: use camelCase

File: assets/airlines/tay.json [hunk 6]
```diff
@@ -20,6 +20,13 @@
         ],
         "b744": [
             ["B744", 4]
+        ],
+        "737": [
```
Comment (Fechulo) [inline assets/airlines/tay.json]: This should be `b737` for consistency

File: assets/airports/airportLoadList.js [hunk 7]
```diff
@@ -375,12 +375,12 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
...
             premium: false
         },
-        // {
-        //     icao: 'loww',
-        //     level: 'medium',
-        //     name: 'Vienna International Airport',
-        //     premium: false
-        // },
+        {
+        icao: 'loww',
```
Comment (Fechulo) [inline assets/airports/airportLoadList.js]: Please indent correctly

File: assets/airlines/azq.json [hunk 8]
```diff
@@ -10,10 +10,13 @@
...
         "default": [
             ["AN12", 3],
-            ["B744", 3],
-            ["B748", 3],
-            ["B763", 2],
+            ["B744", 5],
+            ["B748", 9],
             ["IL76", 7]
+        ],
+        "747": [
```
[the remark this answers is not in the document]
Comment (JonasStuhlberger) [reply assets/airlines/azq.json]: This is a fleet-prefix that applies to both 744s and 748s, so I thought 747 makes more sense, but ok, that is done!

File: assets/airlines/tap.json (hunk 5 above)
Comment (JonasStuhlberger) [reply assets/airlines/tap.json]: Completely removed the whole airbusshort-thing for TAP in the new build, and went with a tap/loww prefix, to get a A320+A321+A332 setup

File: assets/airlines/ezy.json [hunk 9]
```diff
@@ -12,10 +12,10 @@
...
         "default": [
-            ["A20N", 8],
-            ["A319", 82],
-            ["A320", 80],
-            ["A321", 1],
+            ["A20N", 24],
+            ["A319", 125],
+            ["A320", 169],
+            ["A321", 5],
             ["RJ85", 3]
```
[the remark this answers is not in the document]
Comment (Marcel510) [reply assets/airlines/ezy.json]: You are right the RJs should be removed (and airports that used it should be updated) as they were only wetleased and are no longer operated for easyjet. Though I'm not really happy with using wikipedia as source for the airline fleets. Wikipedia is usually fairly outdated. I'd recommend planespotters.net which is more up-to-date. This also one of the sites used by wikipedia contributors to update their data.

File: assets/airlines/ldm.json [hunk 10]
```diff
@@ -4,31 +4,21 @@
...
         "default": [
-            ["A320", 5],
-            ["A321", 4],
-            ["B738", 6]
-        ],
-        "short": [
-            ["A320", 5],
-            ["A321", 4]
-        ],
-        "wetlease": [
```
Comment (Marcel510) [inline assets/airlines/ldm.json]: 1. When removing a fleet from an airline file, please always check if there are any other airports that use this fleet (In this case you removed the b738 fleet which is used by EDDL and might cause errors when playing this airport and the sim wants to load a not existing fleet). Many editors have a "search in folder" function, search the airport folder for "ldm/b738" to see if there are any occurences.

2. The "b738" as well as the "wetlease" fleet should be kept here as laudamotion is a subsiduary of Ryanair Holdings and ryanair operates a couple of wetlease flights with 737s for LDM:
Screenshot_253

Comment (JonasStuhlberger) [reply assets/airlines/ldm.json]: Whoops, yeah I didn't think thru what could happen if I deleted something that is used somewhere else, but thanks for pointing that out in that files. (And I have no idea about the current situation with Ryanair and Laudamotion and I guess Lufthansa in some way too? Super confusing.)
~Done

File: assets/airlines/eth.json [hunk 11]
```diff
@@ -1,50 +1,62 @@
...
-        "b738": [
-            ["B738", 16]
-        ],
-        "b77l": [
-            ["B77L", 6]
-        ],
-        "b77w": [
-            ["B77W", 4]
-        ]
+    {
```
[the remark this answers is not in the document]
Comment (JonasStuhlberger) [reply assets/airlines/eth.json]: ~Corrected

File: assets/airlines/eth.json [hunk 12]
```diff
@@ -1,50 +1,62 @@
...
+        "callsign": {
+            "name": "Ethiopian",
+            "callsignFormats": [
+                "###"
+            ]
+        },
+        "fleets": {
+            "default": [
+                ["A359", 9],
+                ["B737", 7],
```
[the remark this answers is not in the document]
Comment (JonasStuhlberger) [reply assets/airlines/eth.json]: ~Updated the whole ETH fleet, with cargo and their 737 Maxes.

File: documentation/airport-guides/loww.md [hunk 13]
```diff
@@ -0,0 +1,77 @@
...
+295.427 tonnes of cargo
+241.004 aircraft movements
+
+### Airspace
+The Vienna approach is sourrounded by the FIRs of Vienna (LOVV), Praha (LKAA), Bratislava (LZBB) and Budapest (LHCC).
+
+The Airspace located within the LOVV FIR is controlled from SFC to FL245. Wien Radar also controls part of the Budapest FIR, southeast of Vienna. The airspace in this area is controlled by Wien Radar from FL055 to FL245.
+
+### Restricted Airspace
+Aircraft departing from runway 29 make a left turn at 1.000 feet due to restricted airspace LOR-15, which lies over the city of Vienna, and extends up to FL100.
```
Comment (Fechulo) [inline documentation/airport-guides/loww.md]: Use comma instead of period for 1000

File: documentation/airport-guides/loww.md [hunk 14]
```diff
@@ -0,0 +1,77 @@
...
+
+Aircraft arriving into Vienna cross through restricted airspace LOR-15 when approaching runway 11 and 16. IFR flights are allowed to enter this airspace when approaching the airport eastbound or southbound. There are other exceptions to this restriction, including VFR flights in accordance to prescribed procedures or military operations.
+
+### Instrument Procedures
+
+#### Departures
+Aircraft taking off from Vienna follow RNAV SIDs. These SIDs have suffixes corresponding to certain runways.
```
Comment (Fechulo) [inline documentation/airport-guides/loww.md]: Change to:
`Each SID has a suffix that correspond to a specific runway.`

File: documentation/airport-guides/loww.md [hunk 15]
```diff
@@ -0,0 +1,77 @@
...
+| Suffix | Runway          |
+|:------:|:---------------:|
+| A      | 11              |
+| B      | 16              |
+| C      | 29              |
+| D      | 34              |
+| R      | Noise Abatement |
+
+#### Arrivals
+Aircraft descending into Vienna are funneled onto the ILS by RNAV transitions. Each transition has a certain suffix corresponding to the respective runway.
```
Comment (Fechulo) [inline documentation/airport-guides/loww.md]: Change to:
`Each transition has a suffix that corresponds to a specific runway`

File: documentation/airport-guides/loww.md [hunk 16]
```diff
@@ -0,0 +1,77 @@
...
+| U      | 29 Noise Abatement |
+
+### Sectors Available in openScope
+```
+Wien Radar (Departure)
+Wien Radar (Approach)
+Wien Director (Final Approach)
+```
+
+### Other aerodromes within the area of LOWW
```
Comment (Fechulo) [inline documentation/airport-guides/loww.md]: Change to `Airports of Interest`

File: documentation/airport-guides/loww.md [hunk 17]
```diff
@@ -0,0 +1,77 @@
...
+
```
Comment (Fechulo) [inline documentation/airport-guides/loww.md]: LZIB and LOXT should be added to this list

File: documentation/airport-guides/loww.md [hunk 18]
```diff
@@ -4,8 +4,8 @@ Vienna (Wien) Schwechat International Airport is the largest airport in Austria,
 The Airport was originally built in 1938 as a military design and production complex. There was just one runway (11/29), which was expanded in 1959, and a second runway (16/34) was built in 1972. In 2006 a new control tower was opened which allows for overview of the entire airport.
 
 ### Statistics for LOWW, 2018
-27.037.292 passengers
-295.427 tonnes of cargo
+27.037.292 passengers handled
```
Comment (Fechulo) [inline documentation/airport-guides/loww.md]: I wouldn't say 'handled' here

File: assets/airlines/exs.json [hunk 19]
```diff
@@ -15,6 +15,10 @@
             ["B733", 16],
             ["B738", 54],
             ["B752", 11]
+        ],
+        "737": [
```
[the remark this answers is not in the document]
Comment (Fechulo) [reply assets/airlines/exs.json]: This still needs to be corrected

File: assets/airlines/ezy.json (hunk 9 above)
[the remark this answers is not in the document]
Comment (Fechulo) [reply assets/airlines/ezy.json]: This still needs to be addressed

**Comment by Fechulo** [comment]
Continues to be approved and ready to merge now

File: assets/airlines/ezy.json [hunk 20]
```diff
@@ -12,26 +12,22 @@
...
         ],
         "a320": [
-            ["A320", 80]
+            ["A320", 57]
         ],
         "a20n": [
-            ["A20N", 8]
+            ["A20N", 29]
         ],
-        "a321": [
```
Comment (Fechulo) [inline assets/airlines/ezy.json]: Have you checked if this fleet is used elsewhere?

File: assets/airlines/ezy.json [hunk 21]
```diff
@@ -12,26 +12,22 @@
...
-            ["A321", 1]
-        ],
-        "rjxx": [
```
Comment (Fechulo) [inline assets/airlines/ezy.json]: Have you checked if this fleet is used elsewhere?

File: assets/airlines/ldm.json [hunk 22]
```diff
@@ -4,25 +4,26 @@
...
         ]
     },
     "fleets": {
         "default": [
-            ["A320", 5],
+            ["A320", 14],
             ["A321", 4],
             ["B738", 6]
-        ],
-        "short": [
```
Comment (Fechulo) [inline assets/airlines/ldm.json]: Have you checked if this fleet is used elsewhere?

File: assets/airlines/tap.json [hunk 23]
```diff
@@ -31,20 +31,18 @@
             ["A321", 4],
             ["E190", 13]
         ],
-        "airbusShort": [
```
Comment (Fechulo) [inline assets/airlines/tap.json]: Have you checked if this fleet is used elsewhere?

File: documentation/airport-guides/loww.md [hunk 24]
```diff
@@ -0,0 +1,92 @@
...
+The videomap also contains:
+* Top down view of LOAN Wiener Neustadt Ost Airfield
+* Top down view of LOAV Vöslau Köttingbrunn Airfield
+* Top down view of LOXT Tulln Brumowski Air Base
+* All VOR navaids within the controlled airspace
+* 20 mile ILS centerlines
+
+
+### Additional Resources and Charts
+[AustroControl](https://eaip.austrocontrol.at/lo/190719/ad_2_loww.htm)
```
Comment (Fechulo) [inline documentation/airport-guides/loww.md]: This link is broken, I'd suggest just linking to the homepage

File: documentation/airport-guides/loww.md [hunk 25]
```diff
@@ -0,0 +1,92 @@
...
+Runway 10/28, 1067m (3501ft), Asphalt
+
+##### LOAV - Vöslau Köttingbrunn
+Vöslau Köttingbrunn Airfield is located in the town of Köttingbrunn, near Vöslau and lies about 15 miles southwest of Vienna International Airport. It is privately owned and serves general aviation airctaft
+
+Runway 13/31, 950m (3117ft), Asphalt
+Runway 13/31 Grass, 650m (2132ft), Grass
+
+##### LOXT - Tulln Brumowski Air Base
+Brumowski Air Base is located about 3 miles outside the town of Tulln and about 22 miles northeast of Vienna International Airport. It is an Austrian Air Force (German: Österreichische Luftstreitkräft) base and was opened in October of 1939. The base was under American control from 1945 to 1955, and was then over to the Austrian police, since Austria did not have an Air Force at the time.
```
Comment (Fechulo) [inline documentation/airport-guides/loww.md]: was then handed over?

File: assets/airports/loww.json [hunk 26]
```diff
@@ -1,303 +1,589 @@
 {
+    "airac": 1913,
```
Comment (Fechulo) [inline assets/airports/loww.json]: I believe it should all work for airac 2001 too (wow that feels weird)

File: assets/airports/airportLoadList.js [hunk 27]
```diff
@@ -375,12 +375,12 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
...
         },
-        // {
-        //     icao: 'loww',
-        //     level: 'medium',
-        //     name: 'Vienna International Airport',
-        //     premium: false
-        // },
+        {
+            icao: 'loww',
+            level: 'medium',
```
Comment (Fechulo) [inline assets/airports/airportLoadList.js]: If I counted correctly, there are almost 39acph at the moment, which would make this technically 'easy'. You can either change the level to easy or find somewhere to distribute the extra 2.0976acph needed to make this officially 'medium'

**Pull request merged**